bugsnag-android-core/src/main/java/com/bugsnag/android/Bugsnag.java

```diff
@@ -67,9 +67,9 @@ public static Client init(@NonNull Context androidContext, @NonNull Configuratio
     }
 
     private static void logClientInitWarning() {
-        Logger.warn("It appears that Bugsnag.init() was called more than once. Subsequent "
-            + "calls have no effect, but may indicate that Bugsnag is not integrated in an"
-            + " Application subclass, which can lead to undesired behaviour.");
+        DebugLogger.INSTANCE.w("It appears that Bugsnag.init() was called more than once. "
```

Comment:
is `DebugLogger.INSTANCE.w` a sneaky way of calling the logger even when logging is disabled?

Reply:
Yep - this will always be logged. It feels like we would always want to inform the user of this as it would indicate a potential bug in how they have integrated bugsnag.

Reply:
I get that, but what if a user has passed in a custom logger and for some reason they don't see things that go via the Android logger? Seems strange to accept a logger as a configuration option and then bypass it.

Reply:
I have updated this to use the logger on configuration.